lib/api/controllers/funnelController.js

```diff
@@ -119,19 +119,17 @@ function processRequest(kuzzle, controllers, requestObject, context) {
       return requestObject.checkInformation();
     })
     .then(() => {
-      if (!controllers[requestObject.controller] ||
-        !controllers[requestObject.controller][requestObject.action] ||
+      if (!controllers[requestObject.controller] || !controllers[requestObject.controller][requestObject.action] ||
```

Comment:
really little nitpicking, but as we have 3 conditions that are too long to be kept in one single line, I find it more readable to let 1 condition per line.